### ISSUE TITLE
Error in processing quartz-monitor.css and quartz-monitor.js when running 'grails war' (Grails 3.0.4)

### ISSUE THREAD

**Issue by make** [jamescookie/quartz-monitor#32]
This error occurs in processing quartz-monitor.css and quartz-monitor.js when running 'grails war' (Grails 3.0.4)
```
Processing File 46 of 55 - quartz-monitor-main.css
Minifying File 46 of 55 - quartz-monitor-main
Compressing File 46 of 55 - quartz-monitor-main
Processing File 47 of 55 - quartz-monitor-main.js
Uglifying File 47 of 55 - quartz-monitor-main
Compressing File 47 of 55 - quartz-monitor-main
Processing File 48 of 55 - quartz-monitor.css
Error Processing GString Template Engine groovy.lang.MissingPropertyException: No such property: grails for class: groovy.lang.Binding
Error Processing GString Template Engine
groovy.lang.MissingPropertyException: No such property: grails for class: groovy.lang.Binding
	at groovy.lang.Binding.getVariable(Binding.java:56)
	at groovy.lang.Binding.getProperty(Binding.java:103)
	at org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:168)
	at groovy.lang.Closure.getPropertyTryThese(Closure.java:321)
	at groovy.lang.Closure.getPropertyDelegateFirst(Closure.java:311)
	at groovy.lang.Closure.getProperty(Closure.java:296)
	at org.codehaus.groovy.runtime.callsite.PogoGetPropertySite.getProperty(PogoGetPropertySite.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGroovyObjectGetProperty(AbstractCallSite.java:231)
	at groovy.tmp.templates.GStringTemplateScript6$_getTemplate_closure1.doCall(GStringTemplateScript6.groovy:2)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at groovy.lang.Closure.call(Closure.java:423)
	at groovy.lang.Closure$WritableClosure.writeTo(Closure.java:853)
	at groovy.lang.Closure$WritableClosure.toString(Closure.java:979)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:68)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.DirectiveProcessor.directiveArgumentsFromGString(DirectiveProcessor.groovy:172)
	at asset.pipeline.DirectiveProcessor.findDirectives(DirectiveProcessor.groovy:160)
	at asset.pipeline.DirectiveProcessor.getDependencyTree(DirectiveProcessor.groovy:134)
	at asset.pipeline.DirectiveProcessor.compile(DirectiveProcessor.groovy:58)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at asset.pipeline.AssetCompiler.compile(AssetCompiler.groovy:91)
	at asset.pipeline.AssetCompiler$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.gradle.AssetCompile.compile(AssetCompile.groovy:134)
	...
```
```
Processing File 49 of 55 - quartz-monitor.js
Error Processing GString Template Engine groovy.lang.MissingPropertyException: No such property: grails for class: groovy.lang.Binding
Error Processing GString Template Engine
groovy.lang.MissingPropertyException: No such property: grails for class: groovy.lang.Binding
	...
Unable to Locate Asset: /e
Unable to Locate Asset: /quartz-monitor-main
```

build.gradle dependencies:
compile 'org.grails.plugins:asset-pipeline:3.0.8'
compile 'org.grails.plugins:quartz:2.0.0'
compile 'org.grails.plugins:quartz-monitor:1.2'

**Comment by zoran119** [comment]
Same thing happening here.

**Comment by meudaypratap** [comment]
I am also getting the same issue, not able to take this production

**Comment by arjangch** [comment]
I have the same issue. Any solution yet?

**Comment by make** [comment]
With Grails 3.0.10 the error is
```   
Processing File 48 of 55 - quartz-monitor.css
Error Processing GString Template Engine java.lang.NullPointerException: Cannot get property 'quartz' on null object
Error Processing GString Template Engine
java.lang.NullPointerException: Cannot get property 'quartz' on null object
	at org.codehaus.groovy.runtime.NullObject.getProperty(NullObject.java:57)
	at org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:168)
	at org.codehaus.groovy.runtime.callsite.NullCallSite.getProperty(NullCallSite.java:44)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:227)
	at groovy.tmp.templates.GStringTemplateScript11$_getTemplate_closure1.doCall(GStringTemplateScript11.groovy:2)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at groovy.lang.Closure.call(Closure.java:423)
	at groovy.lang.Closure$WritableClosure.writeTo(Closure.java:853)
	at groovy.lang.Closure$WritableClosure.toString(Closure.java:979)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:68)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.DirectiveProcessor.directiveArgumentsFromGString(DirectiveProcessor.groovy:176)
	at asset.pipeline.DirectiveProcessor.findDirectives(DirectiveProcessor.groovy:160)
	at asset.pipeline.DirectiveProcessor.getDependencyTree(DirectiveProcessor.groovy:134)
	at asset.pipeline.DirectiveProcessor.compile(DirectiveProcessor.groovy:58)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at asset.pipeline.AssetCompiler.compile(AssetCompiler.groovy:128)
	at asset.pipeline.AssetCompiler$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.gradle.AssetCompile.compile(AssetCompile.groovy:176)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:63)
	at org.gradle.api.internal.project.taskfactory.AnnotationProcessingTaskFactory$StandardTaskAction.doExecute(AnnotationProcessingTaskFactory.java:218)
...
	at org.gradle.internal.concurrent.DefaultExecutorFactory$StoppableExecutorImpl$1.run(DefaultExecutorFactory.java:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Unable to Locate Asset: /e
Error Processing GString Template Engine java.lang.NullPointerException: Cannot get property 'quartz' on null object
Error Processing GString Template Engine
java.lang.NullPointerException: Cannot get property 'quartz' on null object
	at org.codehaus.groovy.runtime.NullObject.getProperty(NullObject.java:57)
	at org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:168)
	at org.codehaus.groovy.runtime.callsite.NullCallSite.getProperty(NullCallSite.java:44)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:227)
	at groovy.tmp.templates.GStringTemplateScript12$_getTemplate_closure1.doCall(GStringTemplateScript12.groovy:2)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at groovy.lang.Closure.call(Closure.java:423)
	at groovy.lang.Closure$WritableClosure.writeTo(Closure.java:853)
	at groovy.lang.Closure$WritableClosure.toString(Closure.java:979)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:68)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.DirectiveProcessor.directiveArgumentsFromGString(DirectiveProcessor.groovy:176)
	at asset.pipeline.DirectiveProcessor.findDirectives(DirectiveProcessor.groovy:160)
	at asset.pipeline.DirectiveProcessor.getDependencyTree(DirectiveProcessor.groovy:134)
	at asset.pipeline.DirectiveProcessor.compile(DirectiveProcessor.groovy:58)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at asset.pipeline.AssetCompiler.compile(AssetCompiler.groovy:128)
	at asset.pipeline.AssetCompiler$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.gradle.AssetCompile.compile(AssetCompile.groovy:176)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:63)
...
	at org.gradle.internal.concurrent.DefaultExecutorFactory$StoppableExecutorImpl$1.run(DefaultExecutorFactory.java:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Unable to Locate Asset: /e
Minifying File 48 of 55 - quartz-monitor
Compressing File 48 of 55 - quartz-monitor
Processing File 49 of 55 - quartz-monitor.js
Error Processing GString Template Engine java.lang.NullPointerException: Cannot get property 'quartz' on null object
Error Processing GString Template Engine
java.lang.NullPointerException: Cannot get property 'quartz' on null object
	at org.codehaus.groovy.runtime.NullObject.getProperty(NullObject.java:57)
	at org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:168)
	at org.codehaus.groovy.runtime.callsite.NullCallSite.getProperty(NullCallSite.java:44)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:227)
	at groovy.tmp.templates.GStringTemplateScript13$_getTemplate_closure1.doCall(GStringTemplateScript13.groovy:2)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at groovy.lang.Closure.call(Closure.java:423)
	at groovy.lang.Closure$WritableClosure.writeTo(Closure.java:853)
	at groovy.lang.Closure$WritableClosure.toString(Closure.java:979)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:68)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.DirectiveProcessor.directiveArgumentsFromGString(DirectiveProcessor.groovy:176)
	at asset.pipeline.DirectiveProcessor.findDirectives(DirectiveProcessor.groovy:160)
	at asset.pipeline.DirectiveProcessor.getDependencyTree(DirectiveProcessor.groovy:134)
	at asset.pipeline.DirectiveProcessor.compile(DirectiveProcessor.groovy:58)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at asset.pipeline.AssetCompiler.compile(AssetCompiler.groovy:128)
	at asset.pipeline.AssetCompiler$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.gradle.AssetCompile.compile(AssetCompile.groovy:176)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:63)
...
	at org.gradle.internal.concurrent.DefaultExecutorFactory$StoppableExecutorImpl$1.run(DefaultExecutorFactory.java:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Unable to Locate Asset: /e
Error Processing GString Template Engine java.lang.NullPointerException: Cannot get property 'quartz' on null object
Error Processing GString Template Engine
java.lang.NullPointerException: Cannot get property 'quartz' on null object
	at org.codehaus.groovy.runtime.NullObject.getProperty(NullObject.java:57)
	at org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:168)
	at org.codehaus.groovy.runtime.callsite.NullCallSite.getProperty(NullCallSite.java:44)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:227)
	at groovy.tmp.templates.GStringTemplateScript14$_getTemplate_closure1.doCall(GStringTemplateScript14.groovy:2)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at groovy.lang.Closure.call(Closure.java:423)
	at groovy.lang.Closure$WritableClosure.writeTo(Closure.java:853)
	at groovy.lang.Closure$WritableClosure.toString(Closure.java:979)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:68)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.DirectiveProcessor.directiveArgumentsFromGString(DirectiveProcessor.groovy:176)
	at asset.pipeline.DirectiveProcessor.findDirectives(DirectiveProcessor.groovy:160)
	at asset.pipeline.DirectiveProcessor.getDependencyTree(DirectiveProcessor.groovy:134)
	at asset.pipeline.DirectiveProcessor.compile(DirectiveProcessor.groovy:58)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at asset.pipeline.AssetCompiler.compile(AssetCompiler.groovy:128)
	at asset.pipeline.AssetCompiler$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.gradle.AssetCompile.compile(AssetCompile.groovy:176)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:63)
...
	at org.gradle.internal.concurrent.DefaultExecutorFactory$StoppableExecutorImpl$1.run(DefaultExecutorFactory.java:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Unable to Locate Asset: /e
Error Processing GString Template Engine java.lang.NullPointerException: Cannot get property 'quartz' on null object
Error Processing GString Template Engine
java.lang.NullPointerException: Cannot get property 'quartz' on null object
	at org.codehaus.groovy.runtime.NullObject.getProperty(NullObject.java:57)
	at org.codehaus.groovy.runtime.InvokerHelper.getProperty(InvokerHelper.java:168)
	at org.codehaus.groovy.runtime.callsite.NullCallSite.getProperty(NullCallSite.java:44)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:227)
	at groovy.tmp.templates.GStringTemplateScript15$_getTemplate_closure1.doCall(GStringTemplateScript15.groovy:2)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at groovy.lang.Closure.call(Closure.java:423)
	at groovy.lang.Closure$WritableClosure.writeTo(Closure.java:853)
	at groovy.lang.Closure$WritableClosure.toString(Closure.java:979)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.call(PogoMetaMethodSite.java:68)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.DirectiveProcessor.directiveArgumentsFromGString(DirectiveProcessor.groovy:176)
	at asset.pipeline.DirectiveProcessor.findDirectives(DirectiveProcessor.groovy:160)
	at asset.pipeline.DirectiveProcessor.getDependencyTree(DirectiveProcessor.groovy:134)
	at asset.pipeline.DirectiveProcessor.compile(DirectiveProcessor.groovy:58)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at asset.pipeline.DirectiveProcessor$compile.call(Unknown Source)
	at asset.pipeline.AssetCompiler.compile(AssetCompiler.groovy:128)
	at asset.pipeline.AssetCompiler$compile.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at asset.pipeline.gradle.AssetCompile.compile(AssetCompile.groovy:176)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:63)
...
	at org.gradle.internal.concurrent.DefaultExecutorFactory$StoppableExecutorImpl$1.run(DefaultExecutorFactory.java:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Unable to Locate Asset: /e
```

**Comment by jamescookie** [comment]
The only way I can see to get around this issue is to include all the javascript and css, then exclude any that are configured off by modifying the `grails.assets.excludes` config option.

However, I need to use the `require_tree` notation for this, and there seems to be a [bug](https://github.com/bertramdev/grails-asset-pipeline/issues/324) with it. If that gets fixed then I'll be able to fix this.

That said, it doesn't actually seem to affect anything, apart from an ugly looking stacktrace, so I'm not sure why it should stop anyone using the plugin?

**Comment by arjangch** [comment]
In my case, it requires me to give sysadmin a long explanation and re-assurance that everything is fine. Maybe a one hour meeting which is not nice.

**Comment by zyro23** [comment]
getting the same exception @make mentioned. the "fix" pull-request just makes the expression null-safe but does not change the fact that it seems `grails.util.Holders.config` is null during gradle `assetCompile`? until someone comes up with a cleaner solution, cant the jquery assets just be included conditionally directly in the view(s)? e.g.
```
<g:if test="${grailsApplication.config.getProperty("quartz.monitor.showCountdown", Boolean, true}">
    <asset:javascript src="jquery.countdown" />
</g:if>
```

**Comment by jtruty** [comment]
This works on grails 3.1.4: 
//these files give asset pipeline errors, so excluding
`assets.excludes = ["quartz-monitor.css","quartz-monitor.js"]`

**Comment by jamescookie** [comment]
Should be fixed in version 1.3.
I've applied the suggestion from @zyro23.

**Comment by zyro23** [comment]
thank you!
